src/app/pages/features/common/common.html

```diff
@@ -53,6 +75,39 @@ <h2 id="Image" class="link clickable">
   </table>
   <br>
 
+  <h2 id="Source" class="link clickable">
```

Comment:
in my opinion, this interface should be moved inside `features/modal-gallery` because referenced only there.
Also, it's defined in ModalImage object, and not in a common one:
```
export interface ModalImage extends ImageData {
    extUrl?: string;
    downloadFileName?: string;
    sources?: Source[];
}
```

What do you think? Am I missing something?

Reply:
I have nothing against it